javascript-source/games/gambling.js

```diff
@@ -76,7 +76,7 @@
 		 * @commandpath gamblesetmax [amount] - Set how many points people can gamble.
 		 */
 		if (command.equalsIgnoreCase('gamblesetmax')) {
-			if (!action) {
+			if ((!action) || isNaN(action) || (action < 1)) {
```

Comment:
You don't need to add everything in specific parentheses, also since you're updating this change `!action` to `action === undefined`, and use `isNaN(parseInt(action))` something rhino will mess that up.

e.g `((!action) || (action < 1))`